types/human-date/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for human-date 1.4
+// Project: https://github.com/montanaflynn/human-date
+// Definitions by: Rico Sandyca Novenza <https://github.com/ricosandyca>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface PrettyPrintOptions {
+    showTime?: boolean;
+}
+
+export interface RelativeTimeOptions {
+    futureSuffix?: string;
+    pastSuffix?: string;
+    presentText?: string;
+    returnObject?: boolean;
+    allUnits?: boolean;
+}
+
+export interface RelativeTimeReturns {
+    seconds?: number;
+    hours?: number;
+    days?: number;
+    years?: number;
+    past?: boolean;
+}
```

Comment:
I think this is an overshoot here. The object return type is either all-or-nothing here, see, the processing of the results stops of the return type is requested to be an object:
https://github.com/montanaflynn/human-date/blob/master/humandate.js#L67-L74
```suggestion
export interface RelativeTimeReturns {
    seconds: number;
    hours: number;
    days: number;
    years: number;
    past: boolean;
}
```